docs/reference/ilm/overview-ilm.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[testenv="basic"]
+[[overview-index-lifecycle-management]]
+== Index lifecycle management overview
+
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+You can create and apply {ilm-cap} ({ilm-init}) policies to automatically manage your indices 
+according to your performance, resiliency, and retention requirements. 
+Index lifecycle policies can trigger actions such as rollover, shrink, freeze, and delete. 
+
+Typically, you associate a lifecycle policy with an index template so it is applied   
+whenever an index is created from the template. 
+You can also apply a policy manually when you create an index.
+
+Index lifecycle policies are most often used to manage time-series indices. 
+Time-series indices have a four stage lifecycle:
```

Comment:
I think I'd frame this as "ILM is often used to manage hot-cold architecture. Hot-warm-cold architectures are common for time series data, such as logging or metrics."

In any case, the second sentence implies that time-series data always has a hot-cold architecture, which isn't necessarily true. I think it also confuses hot-cold architecture with time-series data. Hot-cold architecture is helpful in contexts outside of time-series data.

I'd also note that hot-cold architecture _can_ include up to four stages. For example, you could easily pull out the `warm` or `delete` stages.